app/features/ai_resistant_assignment_generator/tools.py

```diff
@@ -0,0 +1,31 @@
+# tools.py
+
+def process_files(files: list, verbose: bool = False):
+    """
+    Process uploaded files and convert them into a format usable by the pipeline.
+    
+    Args:
+        files (list): List of uploaded files.
+        verbose (bool): Enable detailed logging if True.
+    
+    Returns:
+        Database object containing processed data.
+    """
+    pipeline = RAGpipeline(verbose=verbose)
```

Comment:
Please, replace the name of this class for a better understanding of its use in the AI-Resistant Assignment context